docs/content/technical-specifications.md

```diff
@@ -19,6 +19,20 @@ The 1.12 release supports the Ingress v1beta1 API and continues to receive secur
 
 We explicitly test the NGINX Ingress Controller on a range of Kubernetes platforms at each release, and the [release notes](/nginx-ingress-controller/releases) list which platforms were tested. We will provide technical support for the NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider and which passes the [Kubernetes conformance tests](https://www.cncf.io/certification/software-conformance/).
 
+{{% table %}}
+| NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
+| --- | --- | --- | --- | --- |
+| 2.2 |  |  |  |  |
```

Comment:
Is this line necessary?

Reply:
2.1.1 will go first, but OK. This won't be a hill I die on 😄

Reply:
you made your point. done.  ;-)